hack/man-page-checker

```diff
@@ -19,7 +19,11 @@ cd $(dirname $0)/../docs/ || die "Please run me from top-level libpod dir"
 
 rc=0
 
-for md in *.1.md;do
+for md in *.1.md; do
+    if [ "$md" = "ramalama-ls.1.md" ]; then
```

Comment:
If the link is not an md file then this is not necessary.

Reply:
Yup that works, thanks for spotting, re-pushed with changes.